LSMS_deflated_admin2.R

```diff
@@ -0,0 +1,256 @@
+welfare_2014 <-read_dta("ECVMA_2014_welfare.dta")
+welfare_2018 <-read_dta("EHCVM_2018_welfare.dta")
+
+
+#slimming down raw 2011 lsms dataset 
+ecvma_slim_2011 <- 
+  welfare_2011 %>% 
+  select(year, hid, 
+         cluster, 
+         fdtexp, #Purchased and auto-consumption food expenditure, nominal (annual)
+         nfdtexp, #Purchased & auto-consumption non-food expenditure, nominal (annual)
+         hhtexp, #Household food and non-food consumption expenditure, nominal (annual)
+         pc_fd, #Per capita food consumption expenditure, nominal (annual)
+         pc_hh, #Per capita food and non-food consumption, nominal (annual)
+         padq_fd, #Per adult equivalent food consumption expenditure, nominal (annual)
+         padq_hh, #Per adult equivalent food and non-food consumption, nominal (annual)
+         pc_fddr, #Per capita food consumption expenditure, deflated (annual)
+         pc_hhdr, #Per capita food and non-food consumption expenditure, deflated (annual)
+         hhsize, 
+         int_year,
+         int_month,
+         latitude,
+         longitude
+  )
+
+#slimming down raw 2014 lsms dataset 
+ecvma_slim_2014 <- 
+  welfare_2014 %>% 
+  select(year, hid, 
+         cluster, 
+         fdtexp, #Purchased and auto-consumption food expenditure, nominal (annual)
+         nfdtexp, #Purchased & auto-consumption non-food expenditure, nominal (annual)
+         hhtexp, #Household food and non-food consumption expenditure, nominal (annual)
+         pc_fd, #Per capita food consumption expenditure, nominal (annual)
+         pc_hh, #Per capita food and non-food consumption, nominal (annual)
+         padq_fd, #Per adult equivalent food consumption expenditure, nominal (annual)
+         padq_hh, #Per adult equivalent food and non-food consumption, nominal (annual)
+         pc_fddr, #Per capita food consumption expenditure, deflated (annual)
+         pc_hhdr, #Per capita food and non-food consumption expenditure, deflated (annual)
+         hhsize, 
+         int_year,
+         int_month,
+         latitude,
+         longitude
+  )
+
+# merge 2011 and 2014 datasets together
+lsms_slim_2011_2014 <- 
+  ecvma_slim_2014 %>% 
+  rbind(ecvma_slim_2011)
+
+#slimming down raw 2018 lsms dataset.Adding year column and renaming latitude and longitude columns
+ecvma_slim_2018 <- 
+  welfare_2018 %>% 
+  select(#year, 
+    hhid, 
+    cluster, 
+    pcexp_food, #bien-être (seulement alimentation)
+    pcexp_nonfood, #bien-être (seulement non-alimentation)
+    GPS__Latitude, 
+    GPS__Longitude, 
+    sh_food,
+    sh_nonfood,
+    cons_pc_real
+  ) %>% 
+  mutate(year = 2018) %>% 
+  rename(latitude = GPS__Latitude, 
+         longitude = GPS__Longitude)
+
+#Converting 2018 food expenditure and total expenditure into deflated values
+
+#Converting from daily to annual by multiplying by 365(Number of days in a year)
+ecvma_slim_2018$cons_pc_real_annual<-ecvma_slim_2018$cons_pc_real*365
```

Comment:
instead of base r, could you work on using tidverse/dyplr eg: mutate. Similarly for next few lines where you are creating new columns